client/core/core.go

```diff
@@ -9510,3 +9513,90 @@ func (c *Core) saveDisabledRateSources() {
 		c.log.Errorf("Unable to save disabled fiat rate source to database: %v", err)
 	}
 }
+
+// DEXLogo returns a dex's logo.
+func (c *Core) DEXLogo(host string) []byte {
+	var cert []byte
+	c.connMtx.RLock()
+	dc, found := c.conns[host]
+	c.connMtx.RUnlock()
+	if found {
+		cert = dc.acct.cert
+	} else {
+		cert = CertStore[c.Network()][host]
+	}
+
+	logoBytes, err := c.fetchDEXLogo(host, cert)
+	if err != nil {
+		c.log.Error("fetchDEXLogo error: %v", err)
+		return nil
+	}
+	return logoBytes
```

Comment:
The http cache would save us from hitting `fetchDEXLogo` too many times if the user is only using one device, and as long as the user's client is sending the headers. If they sign in with multiple devices, or have the cache disabled,   or are using **Core** outside of **webserver**, we would generate repeated requests. We should cache the logo bytes locally.

Reply:
I've added a DEX logo cache in `Core` and also a means to clean up the cache every 24 hours (https://github.com/decred/dcrdex/commit/a3ebb22156e7148b1ad36c401aeeee9b5f392697)